docs/src/main/tut/typeclasses/arrow.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: docs
+title:  "Arrow"
+section: "typeclasses"
+source: "core/src/main/scala/cats/arrow/Arrow.scala"
+scaladoc: "#cats.arrow.Arrow"
+---
+# Arrow
+
+`Arrow` is a useful type class for modeling something that behaves like functions, such as `Function1`: `A => B`, `Kleisli`: `A => F[B]` (also known as `ReaderT`), `Cokleisli`: `F[A] => B`, etc. So useful, that Haskell provides special syntax (the `proc` notation) for composing and combining arrows, similar to the `do` notation for sequencing monadic operations.
+
+To create an `Arrow` instance for a type `F[A, B]`, the following abstract methods need to be implemented:
```

Comment:
It might be helpful to show a few examples of using `Arrow` methods on instances that already exist (such as `Function1` before diving into what's needed to create your own custom `Arrow` instance. It's very rare that you'll be creating your own custom `Arrow` instance, but more common that people will want to take advantage of some of the methods on existing types.